src/app/src/main/kotlin/org/imperial/mrc/hint/models/Response.kt

```diff
@@ -1,18 +1,20 @@
 package org.imperial.mrc.hint.models
 
 import com.fasterxml.jackson.databind.ObjectMapper
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
 
 data class SuccessResponse(val data: Any?){
-    val errors = mapOf<Any, Any>()
+    val errors = listOf<ErrorDetail>()
```

Comment:
corresponds to the update schema here: https://github.com/mrc-ide/hintr/pull/23/files#diff-b13e27c6bcde278d556bb9d21fd628a0